themes/classic/_dev/css/components/imageslider.scss

```
@@ -22,6 +22,28 @@
       max-width: 340px;
       .caption-description p {
         color: white;
+        @media (max-width: 767px) {
+          font-size: .875rem;
+        }
+        @media (max-width: 480px){
```

```suggestion
        @include media-breakpoint-down(sm) {
```

But watch out, `sm` is 576px

---

Further is this document we can read:

```
@include media-breakpoint-down(md) {
  .carousel {
    .carousel-item {

[...]

      .caption-description {
        display: none;
      }
    }
  }
}
```

No need to redeclare it, it is already hidden.